rx_signal_processing/dsp.hpp

```diff
@@ -18,40 +29,41 @@ inline void throw_on_cuda_error(cudaError_t code, const char *file, int line)
     std::stringstream ss;
     ss << file << "(" << line << ")";
     std::string file_and_line;
-    ss >> file_and_line; // REVIEW #25 is this necessary to have stringstream then string vs just putting it in a string?
-                         // REPLY many ways to do this. stringstream lets you use >> << though.
-    throw thrust::system_error(code, thrust::cuda_category(), file_and_line); // REVIEW #6 Should we be catching all the thrown cuda errors?
-                                                                              // REPLY I planned for that, but only if we can recover from it. These signify pretty critical errors.
+    ss >> file_and_line;
+    throw thrust::system_error(code, thrust::cuda_category(), file_and_line);
   }
 }
 
 std::vector<cudaDeviceProp> get_gpu_properties();
+void print_gpu_properties(std::vector<cudaDeviceProp> gpu_properties);
+
 
+/**
+ * @brief      Contains the core DSP work done on the GPU.
+ */
 class DSPCore {
```

Comment:
was just looking at this some more and noticed call_decimate is gone but still used in rx_dsp_chain - just curious why you removed it